README.md

```diff
@@ -33,18 +33,30 @@ Example file to place at `~/.puppetlabs/client-tools/puppetdb.conf`:
 ~~~json
 
     {
-        "default_environment":"prod",
-        "environments":{
-            "prod":{
-                "server_urls":[ "https://alpha-rho.local:8081" ],
-                "ca":"<path to ca.pem>",
-                "cert":"<path to cert .pem>",
-                "key":"<path to private-key .pem>"
-            },
-            "dev":{
-                "server_urls":[ "http://localhost:8080" ]
-            }
-        }
+       "puppetdb":{
+                     "server_urls":"https://alpha-rho.local:8081",
+                     "ca":"<path to ca.pem>",
+                     "cert":"<path to cert .pem>",
+                     "key":"<path to private-key .pem>"
+                  }
```

Comment:
@senior I know you can up with the other format, you have any problems aligning this config format with the other tools (sync/terminus/console)